configs/default/PYDR-PYRODRONEF4PDB.config

```diff
@@ -0,0 +1,106 @@
+# Betaflight / STM32F405 (S405) 4.1.1 Nov 15 2019 / 12:54:53 (1e5e3d369) MSP API: 1.42
+
+board_name PYRODRONEF4PDB
+manufacturer_id PYDR
+
+# resources
+resource BEEPER 1 B05
+resource MOTOR 1 B01
+resource MOTOR 2 B00
+resource MOTOR 3 C09
+resource MOTOR 4 A08
+resource LED_STRIP 1 B08
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 B10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 5 C12
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 B11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 5 D02
+resource SERIAL_RX 6 C07
+resource INVERTER 1 C03
+resource LED 1 B04
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource ESCSERIAL 1 B09
+resource CAMERA_CONTROL 1 B09
+resource ADC_BATT 1 C02
+resource ADC_CURR 1 C01
+resource PINIO 1 A15
+resource PINIO 2 C15
+resource OSD_CS 1 B12
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 A04
+resource BARO_CS 1 C14
+resource FLASH_CS 1 C13
+resource I2C_SCL 2 B10
+resource I2C_SDA 2 B11
+
+# timer
+# pin B09: TIM11 CH1 (AF3)
+timer B09 AF3           
+# pin B01: TIM3 CH4 (AF2)
+timer B01 AF2           
+# pin B00: TIM8 CH2N (AF3)
+timer B00 AF3          
+# pin C09: TIM8 CH4 (AF3)
+timer C09 AF3          
+# pin A08: TIM1 CH1 (AF1)
+timer A08 AF1           
+# pin B08: TIM4 CH3 (AF2)
+timer B08 AF2           
+
+# dma
+# ADC 1: DMA2 Stream 4 Channel 0
+dma ADC 1 1             
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin B01 0           
+# pin B00: DMA2 Stream 2 Channel 0
+dma pin B00 0           
+# pin C09: DMA2 Stream 7 Channel 7
+dma pin C09 0           
+# pin A08: DMA2 Stream 1 Channel 6
+dma pin A08 1           
+# pin B08: DMA1 Stream 7 Channel 2
+dma pin B08 0           
+
+# feature
+feature OSD
+
+# aux
+# turn on vtx PIT switch 
+aux 0 40 255 900 2100 0 0           
+# turn on BT module
+aux 1 41 255 900 2100 0 0               
+
+# serial
+# set up for BT module
+serial 3 1 19200 57600 0 115200       
+
+
+# master
+set mag_i2c_device = 2
+set baro_spi_device = 2
+set blackbox_device = SPIFLASH
+set flash_spi_bus = 2
+set max7456_spi_bus = 2
+set motor_pwm_protocol = DSHOT600
```

Comment:
If this setting is used as a default, and if your board is used in combination with older ESCs that do not support Dshot, this will result in the motors spinning up as soon as a battery is connected, potentially resulting in damage or injury.
Do you assume all responsibility for personal injury and damage resulting from using this default setting?

Reply:
Ok. Sorry my mistake. I will take that out. Since this is with PDB. People may have some old esc laying around to use. I change that.